_posts/2020-10-22-modele-wdrozenia-w-chmurze.md

```diff
@@ -0,0 +1,199 @@
+---
+layout:    post
+title:     "W chmurze czyli jak? O możliwych kierunkach rozwoju aplikacji chmurowych"
+date:      2020-08-27 08:00:00 +0100
+published: true
+author:    glipecki
+image:     /assets/img/posts/2020-10-22-modele-wdrozenia-w-chmurze/cloud.jpg
+tags:
```

Comment:
To już Jacek pisał, ale faktycznie przydałoby się wincyj tagów, żeby się fajnie posty podpowiadały 😎